src/encoder.rs

```diff
@@ -1745,6 +1824,19 @@ pub fn encode_block_post_cdef<T: Pixel>(
     }
   }
 
+  if fi.sequence.enable_intra_edge_filter {
+    for y in 0..bsize.height_mi() {
```

Comment:
It could be make simpler if coded_block_info is a `Box<[]>` instead of a `Vec<Vec<>>`.

Reply:
Changed to a new structure with a single vector and `Index`/`IndexMut` implementations instead.